app/design/frontend/Magento/elogic-theme/web/css/source/_variables.less

```diff
@@ -20,6 +28,19 @@
 @font-family-name__base: @font-family-poppins;
 @font-family__base: @font-family-name__base, @font-family__sans-serif;
 
+//  Responsive breakpoints
+
+@screen__m_s: 480px;
+@screen__m_l: 767px;
```

Comment:
You don't need 767px breakpoint, you can check here (`(@screen__m - 1) ` uses for max-width): 
code/lib/web/css/source/lib/_responsive.less